composer.json

```diff
@@ -1,23 +1,63 @@
 {
-  "name": "",
```

Comment:
Here i think , we can fill the name of our app. so we can put "name : 'ImNotFoolJustSick'